addon/services/render-states.js

```diff
@@ -75,10 +80,11 @@ export default Service.extend(Evented, {
     @public
   */
   resetRenderState() {
+    let scheduledCalls = get(this, 'scheduledCalls');
+    Object.values(scheduledCalls).forEach(call => cancel(call));
+    this._resetProperties();
     setProperties(this, {
-      renderQueue: {},
       maxRenderPriority: MAX_RENDER_PRIORITY,
-      availablePriorities: A(),
       // We need't trigger state change for reset. It should be handled through the context change.
       renderState: CRITICAL_RENDER_STATE 
```

Comment:
These can also be moved to the _resetProperties method as they are the defaults.

Reply:
Thanks for pointing out, have accommodated in latest commit